### PR TITLE
Handle tags shorter than "diff." without an exception

### PR DESCRIPTION
**Pull request by jfehrle**
**Kind:** bug fix

### CODE REVIEW

File: vernac/topfmt.ml [hunk 1]
```diff
@@ -201,7 +201,7 @@ let set_emacs_print_strings () =
   let diff = "diff." in
   List.iter (fun b ->
       let (name, attrs) = b in
-      if diff = (String.sub name 0 (String.length diff)) then
+      if diff = (String.sub name 0 (min (String.length name) (String.length diff))) then
```
Comment (SkySkimmer) [inline vernac/topfmt.ml]:
```suggestion
      if CString.is_sub diff name 0 then
```
?

Comment (jfehrle) [reply vernac/topfmt.ml]: Got it, thanks.

**Comment by jfehrle** [comment]
@SkySkimmer, I took your suggestion.  Can you review and merge this or does it have to be @mattam82?

**Review by SkySkimmer — APPROVED** [review]

**Comment by jfehrle** [comment]
Thank you, sir!

**Pull request merged**